src/framework-support/frameworks/corespring-legacy.js

```diff
@@ -6,6 +6,10 @@ export function support(dependencies) {
 
   if (isLegacy) {
     return {
+      externals: {
+        js: ['https://cdn.mathjax.org/mathjax/2.7-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'],
+        css: []
```

Comment:
can we make it optional? no need to add an empty array - treat that as the default.